### PR TITLE
Make replaceTrack() handle "not sending yet" case (fixes: #438)

### PR DESCRIPTION
**Pull request by adam-be**

### CODE REVIEW

**Comment by adam-be** [comment]
Proposed fix for issue #438

File: webrtc.html [hunk 1]
```diff
@@ -3483,6 +3494,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
...
+                <var>connection</var>'s <code><a href=
+                "#dom-peerconnection-currentlocaldesc"
+                >currentLocalDescription</a></code>.</p>
+              </li>
+
+              <li>
+                <p>If <var>sender</var> is not represented in
+                <var>localDescription</var>, then set <var>sender</var>'s
+                <code><a href="#dom-rtpsender-track">track</a></code> attribute
+                to <var>withTrack</var>, and resolves <var>p</var> with
```
Comment (alvestrand) [inline webrtc.html]: Nit: resolves -> resolve (the sentence is in active voice).

File: webrtc.html [hunk 2]
```diff
@@ -3496,16 +3524,12 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                     Ignore which <code>MediaStream</code> the track resides in and
                     the <code>id</code> attribute of the track in this
                     determination. If negotiation is needed, then reject
-                    <var>p</var> with <code>InvalidModificationError</code>, return
-                    <var>p</var> and abort these steps.</p>
+                    <var>p</var> with <code>InvalidModificationError</code>.</p>
```
Comment (alvestrand) [inline webrtc.html]: I think "and abort these steps" needs to be retained.

**Comment by alvestrand** [comment]
I think I've commented on what needs commenting.

File: webrtc.html (hunk 2 above)
Comment (adam-be) [inline webrtc.html]: True. It was only the return I intended to remove.

**Pull request merged**